economics.toml

```diff
@@ -24,24 +24,24 @@
     LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
     DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
     ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
-    ProtocolSustainabilityAddress = "erd135enn78u7fy352ny9mcl26a08wvu02mjdumfkt7cn587cwtnerksagqsfm"
+    ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
     TopUpGradientPoint = "3000000000000000000000000" # 3MIL eGLD (eligible topUp)
     TopUpFactor = 0.25 # fraction of value 0.25 - 25%
 
     [[RewardsSettings.RewardsConfigByEpoch]]
-    EpochEnable = 326
+    EpochEnable = 1
     LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
     DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
     ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
-    ProtocolSustainabilityAddress = "erd135enn78u7fy352ny9mcl26a08wvu02mjdumfkt7cn587cwtnerksagqsfm"
+    ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
     TopUpGradientPoint = "2000000000000000000000000" # 2MIL eGLD (eligible topUp)
     TopUpFactor = 0.5 # fraction of value 0.5 - 50%
 
 [FeeSettings]
     GasLimitSettings = [
```

Comment:
Maybe use just one value for epoch 0. And probably needs more adjustments

Reply:
Changed with only one entry in gas settings for round time of 1 sec